src/frontends/pytorch/src/op_table.cpp

```diff
@@ -326,13 +325,11 @@ OP_CONVERTER(translate_sum_fx);
 OP_CONVERTER(translate_std_fx);
 OP_CONVERTER(translate_topk_fx);
 OP_CONVERTER(translate_to_fx);
-OP_CONVERTER(translate_transpose_fx);
 OP_CONVERTER(translate_quantize_per_channel_fx);
 OP_CONVERTER(translate_quantize_per_tensor_fx);
 OP_CONVERTER(translate_var_fx);
 OP_CONVERTER(translate_var_mean_fx);
 OP_CONVERTER(translate_unbind_int_fx);
-OP_CONVERTER(translate_unique2);
```

Comment:
why is it changes?

Reply:
It is a leftover from other changes. No needed anymore